src/main/resources/default/taglib/t/sidebar.html.pasta

```diff
@@ -15,10 +15,9 @@
             </div>
         </div>
     </div>
-    <div class="col-12 col-lg-8 col-xl-9 d-flex">
-        <div class="flex-grow-1">
+    <div class="col-12 col-lg-8 col-xl-9">
+        <div class="d-flex flex-column flex-grow-1 h-100">
```

Comment:
```suggestion
        <div class="d-flex flex-column h-100">
```
Flex Grow property without a flex container is pretty pointless and will be ignored by the browser